[2025-02-10T12:45:08.468+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T12:45:08.501+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [queued]>
[2025-02-10T12:45:08.512+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [queued]>
[2025-02-10T12:45:08.512+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T12:45:08.524+0000] {taskinstance.py:2889} INFO - Executing <Task(ExternalTaskSensor): wait_for_tests_to_run> on 2025-02-10 12:43:24.873249+00:00
[2025-02-10T12:45:08.542+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=347) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T12:45:08.544+0000] {standard_task_runner.py:72} INFO - Started process 362 to run task
[2025-02-10T12:45:08.544+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cluster_deployment_rollback', 'wait_for_tests_to_run', 'manual__2025-02-10T12:43:24.873249+00:00', '--job-id', '596', '--raw', '--subdir', 'DAGS_FOLDER/cluster_deployment_rollback.py', '--cfg-path', '/tmp/tmpkwppzz83']
[2025-02-10T12:45:08.545+0000] {standard_task_runner.py:105} INFO - Job 596: Subtask wait_for_tests_to_run
[2025-02-10T12:45:08.608+0000] {task_command.py:467} INFO - Running <TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [running]> on host 1999c8c59930
[2025-02-10T12:45:08.730+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cluster_deployment_rollback' AIRFLOW_CTX_TASK_ID='wait_for_tests_to_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T12:43:24.873249+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-10T12:43:24.873249+00:00'
[2025-02-10T12:45:08.733+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T12:45:08.733+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T12:45:08.734+0000] {logging_mixin.py:190} INFO - Current task name:wait_for_tests_to_run state:running start_date:2025-02-10 12:45:08.502603+00:00
[2025-02-10T12:45:08.734+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment_rollback and current dag run status:running
[2025-02-10T12:45:08.734+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T12:45:08.735+0000] {baseoperator.py:421} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-02-10T12:45:08.752+0000] {external_task.py:258} INFO - Poking for tasks ['tests_finished'] in dag run_e2e_tests on 2025-02-10T12:43:25.873249+00:00 ... 
[2025-02-10T12:45:08.754+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:97: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2025-02-10T12:45:08.811+0000] {taskinstance.py:309} INFO - Rescheduling task, marking task as UP_FOR_RESCHEDULE
[2025-02-10T12:45:08.813+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T12:45:08.844+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T12:45:08.874+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-10T12:45:08.876+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-10T12:45:40.404+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T12:45:40.415+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [queued]>
[2025-02-10T12:45:40.420+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [queued]>
[2025-02-10T12:45:40.420+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T12:45:40.427+0000] {taskinstance.py:2889} INFO - Executing <Task(ExternalTaskSensor): wait_for_tests_to_run> on 2025-02-10 12:43:24.873249+00:00
[2025-02-10T12:45:40.433+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=413) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T12:45:40.435+0000] {standard_task_runner.py:72} INFO - Started process 414 to run task
[2025-02-10T12:45:40.436+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cluster_deployment_rollback', 'wait_for_tests_to_run', 'manual__2025-02-10T12:43:24.873249+00:00', '--job-id', '601', '--raw', '--subdir', 'DAGS_FOLDER/cluster_deployment_rollback.py', '--cfg-path', '/tmp/tmp8hzh660m']
[2025-02-10T12:45:40.437+0000] {standard_task_runner.py:105} INFO - Job 601: Subtask wait_for_tests_to_run
[2025-02-10T12:45:40.479+0000] {task_command.py:467} INFO - Running <TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [running]> on host 1999c8c59930
[2025-02-10T12:45:40.524+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cluster_deployment_rollback' AIRFLOW_CTX_TASK_ID='wait_for_tests_to_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T12:43:24.873249+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-10T12:43:24.873249+00:00'
[2025-02-10T12:45:40.527+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T12:45:40.527+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T12:45:40.527+0000] {logging_mixin.py:190} INFO - Current task name:wait_for_tests_to_run state:running start_date:2025-02-10 12:45:40.415986+00:00
[2025-02-10T12:45:40.528+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment_rollback and current dag run status:running
[2025-02-10T12:45:40.528+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T12:45:40.529+0000] {baseoperator.py:421} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-02-10T12:45:40.548+0000] {external_task.py:258} INFO - Poking for tasks ['tests_finished'] in dag run_e2e_tests on 2025-02-10T12:43:25.873249+00:00 ... 
[2025-02-10T12:45:40.553+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:97: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2025-02-10T12:45:40.578+0000] {taskinstance.py:309} INFO - Rescheduling task, marking task as UP_FOR_RESCHEDULE
[2025-02-10T12:45:40.578+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T12:45:40.611+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T12:45:40.630+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-10T12:45:40.631+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-10T12:46:13.927+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-10T12:46:13.941+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [queued]>
[2025-02-10T12:46:13.950+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [queued]>
[2025-02-10T12:46:13.951+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-10T12:46:13.960+0000] {taskinstance.py:2889} INFO - Executing <Task(ExternalTaskSensor): wait_for_tests_to_run> on 2025-02-10 12:43:24.873249+00:00
[2025-02-10T12:46:13.968+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=444) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-10T12:46:13.969+0000] {standard_task_runner.py:72} INFO - Started process 450 to run task
[2025-02-10T12:46:13.970+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cluster_deployment_rollback', 'wait_for_tests_to_run', 'manual__2025-02-10T12:43:24.873249+00:00', '--job-id', '608', '--raw', '--subdir', 'DAGS_FOLDER/cluster_deployment_rollback.py', '--cfg-path', '/tmp/tmply4z7tkr']
[2025-02-10T12:46:13.972+0000] {standard_task_runner.py:105} INFO - Job 608: Subtask wait_for_tests_to_run
[2025-02-10T12:46:14.010+0000] {task_command.py:467} INFO - Running <TaskInstance: cluster_deployment_rollback.wait_for_tests_to_run manual__2025-02-10T12:43:24.873249+00:00 [running]> on host 1999c8c59930
[2025-02-10T12:46:14.127+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cluster_deployment_rollback' AIRFLOW_CTX_TASK_ID='wait_for_tests_to_run' AIRFLOW_CTX_EXECUTION_DATE='2025-02-10T12:43:24.873249+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-10T12:43:24.873249+00:00'
[2025-02-10T12:46:14.139+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-10T12:46:14.139+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-10T12:46:14.140+0000] {logging_mixin.py:190} INFO - Current task name:wait_for_tests_to_run state:running start_date:2025-02-10 12:46:13.942525+00:00
[2025-02-10T12:46:14.140+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment_rollback and current dag run status:running
[2025-02-10T12:46:14.141+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-10T12:46:14.142+0000] {baseoperator.py:421} WARNING - ExternalTaskSensor.execute cannot be called outside TaskInstance!
[2025-02-10T12:46:14.168+0000] {external_task.py:258} INFO - Poking for tasks ['tests_finished'] in dag run_e2e_tests on 2025-02-10T12:45:53.753457+00:00 ... 
[2025-02-10T12:46:14.169+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/session.py:97: DeprecationWarning: This method is deprecated and will be removed in future.
  return func(*args, session=session, **kwargs)

[2025-02-10T12:46:14.179+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-02-10T12:46:14.189+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-10T12:46:14.190+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cluster_deployment_rollback, task_id=wait_for_tests_to_run, run_id=manual__2025-02-10T12:43:24.873249+00:00, execution_date=20250210T124324, start_date=20250210T124613, end_date=20250210T124614
[2025-02-10T12:46:14.199+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-10T12:46:14.199+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-10T12:46:14.200+0000] {logging_mixin.py:190} INFO - Dag name:cluster_deployment_rollback queued_at:2025-02-10 12:43:24.927822+00:00
[2025-02-10T12:46:14.202+0000] {logging_mixin.py:190} INFO - Task hostname:1999c8c59930 operator:ExternalTaskSensor
[2025-02-10T12:46:14.230+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-10T12:46:14.270+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-10T12:46:14.272+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
